design-week/learning-outcomes.md

```diff
@@ -0,0 +1,29 @@
+# Design Week Learning Outcomes
+
+### UX and Design
```

Comment:
Not a blocker but this should be an `h2` (unless there's a missing heading that should be above it)

```suggestion
## UX and Design
```